ROCCurvePlot.py

```diff
@@ -480,27 +348,6 @@
 Run3_TauEff_legend.Draw()
 can.SaveAs("Run3_TauEfficiency.pdf")
 
-
-#FakeRate = ROOT.TMultiGraph("fakerate", "fakerate")
-#FakeRate_legend = ROOT.TLegend(0.7, 0.5, 0.88, 0.65)
-#Run2_FakeRate = ROOT.TGraph(100, scoreSpace, Run2_ROC["FakeRate"])
-#Run3_FakeRate = ROOT.TGraph(100, scoreSpace, Run3_ROC["FakeRate"])
-#Run2_FakeRate.SetLineColor(ROOT.kRed)
-#Run3_FakeRate.SetLineColor(ROOT.kBlue)
-##Run2_FakeRate.SetMaximum(1)
-#FakeRate.Add(Run2_FakeRate)
-#FakeRate.Add(Run3_FakeRate)
-#FakeRate_legend.AddEntry(Run2_FakeRate, "Run2_TT_13TeV")
-#FakeRate_legend.AddEntry(Run3_FakeRate, "Run3_TT_13.6TeV")
-#FakeRate.SetTitle("Fake Rate for Run2 and Run3 TT backgrounds; score; Fake Rate")
-#FakeRate.Draw("A")
-#FakeRate_legend.Draw()
-#can.SaveAs("FakeRate.pdf")
-#
-#ROC_legend = ROOT.TLegend(0.6, 0.6, 0.88, 0.85)
-#ROC_legend.SetBorderSize(0)
```

Comment:
Keep these lines. They plot the fake rate, which is part of the ROC curve. I just commented out to look at some specific plots before.

Reply:
[Line 489](https://github.com/dally96/DisplacedTauAnalysis/pull/25/files#diff-03b723c0888b0b3a1e05b700929c2a3d5fc651a0b1e625de2f8523b2ddb1f8c2L489) included?